st2common/bin/st2ctl

```diff
@@ -14,14 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-LOGFILE="/dev/null"
 COMPONENTS="st2actionrunner st2api st2stream st2auth st2garbagecollector st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2chatops mistral"
-STANCONF="/etc/st2/st2.conf"
+ST2_CONFIG="/etc/st2/st2.conf"
```

Comment:
I was going to complain about this since we use `STANCONF` in other places...but we also use `ST2_CONF` in other places and I prefer the name `ST2_CONF` so, YOLO?